setuptools/command/bdist_wheel.py

```diff
@@ -291,10 +291,15 @@ def _validate_py_limited_api(self) -> None:
             raise ValueError(f"py-limited-api must match '{PY_LIMITED_API_PATTERN}'")
 
         if sysconfig.get_config_var("Py_GIL_DISABLED"):
+            # sys.abiflags is only defined on POSIX
+            if getattr(sys, "abiflags", ""):
```

Comment:
I assume you just mean to check that the attribute exists, and not the truthiness of its value.

```suggestion
            if hasattr(sys, "abiflags"):
```